source/exploring_catalog_and_datasets/04_getting_involved/exporting_data.rst

```diff
@@ -4,44 +4,68 @@ Exporting data
 .. image:: images/export.png
    :alt: Export a dataset
 
-Opendatasoft datasets can be downloaded in various formats. Standard formats are always available for any dataset. Other more specific formats can be available, but they depend on the nature of the data (that is, if the dataset contains geographical coordinates or not).
+Opendatasoft datasets can be downloaded in various formats.
+Flat file formats are always available for any dataset.
+More specific formats can be available, depending on whether the dataset contains geographical coordinates or not.
 
-In the Export tab of a dataset, the following formats can be available:
+1. Go to the **Export** tab of the dataset.
+2. Choose the file format to download.
+3. Perform one of the following actions:
 
-- CSV
-- JSON
-- Excel
-- GeoJSON
-- Shapefile
-- KML
+   * Click **Whole dataset** next to the desired format.
+   * If the dataset is filtered and you want to export the filtered data only, click **Only the selected records** next to the desired format.
 
 .. admonition:: Note
    :class: note
 
-   It is possible to download the dataset in other formats using the `Opendatasoft Search API <https://help.opendatasoft.com/apis/ods-search-v2/#exporting-records>`_.
+   - You can use the parameter :code:`use_labels_for_header=true` in the exports links to use the labels instead of the technical identifiers of the fields as a header (not available in the Opendatasoft Search API v2).
+   - You can also export datasets in other formats using the `Opendatasoft Search API v2 <https://help.opendatasoft.com/apis/ods-search-v2/#exporting-records>`_.  
 
-.. admonition:: Note
-   :class: note
+Available export formats
+------------------------
 
-   There are 2 limitations in Shapefile exports.
+In the Export tab of a dataset, the following formats can be available:
 
-   - The export cannot contain more than 50,000 records.
-   - The export cannot contain more than one geographical shape type (e.g. point, polygon, linestring etc.). Only the first type of the first non-empty record will be kept in the export.
+Flat file formats
+~~~~~~~~~~~~~~~~~
 
-1. Go to the Export tab of the dataset.
-2. Choose the file format to download.
-3. Click on "Whole dataset" next to the chosen format.
+.. list-table::
+   :header-rows: 1
+   :widths: 25 75
+
+   * * Format
+     * Notes
+   * * CSV
+     * CSV uses semicolon (;) as a separator. 
+   * * JSON
+     * 
+   * * Excel
```

Comment:
Maybe something to add with regards to our Excel output is the type of Excel, I know it has led to some problems in the past (due to Excel 2003 vs. 2007 vs. 2010…). In the ticket https://secure.helpscout.net/conversation/1443040226/18380/ , @NTerpo explained that it was in fact "a Microsoft Office XML 2003 with an XLS extension"… I don’t know if we want to publicly document it, though

Reply:
Imho we don't want to publicly document it for now. Let's wait for the work on the download / exports to see if that change anything on this topic and re-discuss it at that moment.